hack/update-openapi-spec.sh

```diff
@@ -107,7 +107,7 @@ mkdir -p "${OPENAPI_ROOT_DIR}/v3"
 # ".well-known__openid-configuration_openapi.json"
 rm -r "${OPENAPI_ROOT_DIR}"/v3/{*,.*} || true
 
-curl -w "\n" -kfsS -H 'Authorization: Bearer dummy_token' "https://${API_HOST}:${API_PORT}/openapi/v3" | jq '.Paths' | jq -r '.[]' | while read -r group; do
+curl -w "\n" -kfsS -H 'Authorization: Bearer dummy_token' "https://${API_HOST}:${API_PORT}/openapi/v3" | jq -r '.Paths | to_entries | .[].key' | while read -r group; do
```

Comment:
curious why this change was needed

Reply:
The format of `/openapi/v3` has changed. The change mirrors that of the removed `gvList` you commented on later in this PR.